package.json

```diff
@@ -19,7 +19,7 @@
     "build:digital-payments": "yarn --cwd integration-libs/digital-payments run build:schematics && ng build digital-payments --configuration production",
     "build:epd-visualization": "yarn --cwd integration-libs/epd-visualization run build:schematics && ng build epd-visualization --configuration production",
     "build:order": "yarn --cwd feature-libs/order run build:schematics && ng build order --configuration production",
-    "build:libs": "ng build core --configuration production && ng build storefrontlib --configuration production && yarn build:schematics && yarn build:user && yarn build:cart && yarn build:order && yarn build:checkout && yarn build:asm && yarn build:tracking  && concurrently --kill-others-on-fail -m 4 yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:smartedit yarn:build:qualtrics yarn:build:assets yarn:build:cdc yarn:build:cds yarn:build:digital-payments yarn:build:epd-visualization && yarn build:setup",
+    "build:libs": "ng build core --configuration production && ng build storefrontlib --configuration production && yarn build:schematics && yarn build:user && yarn build:cart && yarn build:order && yarn build:checkout && yarn build:asm && yarn build:tracking  && concurrently --kill-others-on-fail -m 4 yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:smartedit yarn:build:qualtrics yarn:build:assets yarn:build:cdc yarn:build:cds yarn:build:digital-payments yarn:build:epd-visualization && yarn build:setup && yarn build:commerce-quotes",
```

Comment:
After some thinking, I would put it before we build setup lib (or maybe even remove `build:setup` as it's obsolete library) and put it right after visualization lib